data/brands/amenity/fast_food.json

```
@@ -4404,6 +4409,7 @@
         "brand:wikidata": "Q752941",
         "brand:wikipedia": "en:Taco Bell",
         "cuisine": "tex-mex",
+        "diet:vegetarian": "yes",
```

```suggestion
        "diet:vegan": "yes",
```
Taco Bell also has an [offical path](https://www.tacobell.com/blog/how-to-eat-vegan) to modifying menu options to make them vegan.